src/dev/sdram.cpp

```diff
@@ -492,15 +514,15 @@ void HAL_SDRAM_MspDeInit(SDRAM_HandleTypeDef *sdramHandle)
 
 void __attribute__((constructor)) SDRAM_Init()
 {
-    //	extern void *_sisdram_data, *_ssdram_data, *_esdram_data;
-    //	extern void *_ssdram_bss, *_esdram_bss;
+    //    extern void *_sisdram_data, *_ssdram_data, *_esdram_data;
+    //    extern void *_ssdram_bss, *_esdram_bss;
 
-    //	void **pSource, **pDest;
-    //	for (pSource = &_sisdram_data, pDest = &_ssdram_data; pDest != &_esdram_data; pSource++, pDest++)
-    //		*pDest = *pSource;
+    //    void **pSource, **pDest;
+    //    for (pSource = &_sisdram_data, pDest = &_ssdram_data; pDest != &_esdram_data; pSource++, pDest++)
+    //        *pDest = *pSource;
     //
-    //	for (pDest = &_ssdram_bss; pDest != &_esdram_bss; pDest++)
-    //		*pDest = 0;
+    //    for (pDest = &_ssdram_bss; pDest != &_esdram_bss; pDest++)
+    //        *pDest = 0;
 }
 
 //Program the <project name>_QSPIFLASH.bin file to QSPIFLASH manually
```

Comment:
Is this used/needed? if not lets clean it up.

Reply:
As far as I can tell, it's not used at all.